Observer/Admin/DashboardInstantiation.php

```diff
@@ -140,7 +140,13 @@ public function execute(Observer $observer) : void
                 'platformVersion' => (string) $this->productMetadata->getVersion()
             ];
 
-            $installResult = InstallerClient::install('magento', $installRequestData);
+            // phpcs:ignore
+            $devToken = getenv('NS8_DEV_TOKEN');
+            $installResult = [ 'accessToken' => $devToken ];
+            if (!$devToken) {
+                $installResult = InstallerClient::install('magento', $installRequestData);
+            }
```

Comment:
I like this a lot, but maybe the environment variable should be named `NS8_ACCESS_TOKEN` or something a bit less generic.